src/bootstrap/compile.rs

```diff
@@ -306,7 +306,7 @@ fn copy_self_contained_objects(
             .unwrap_or_else(|| {
                 panic!("Target {:?} does not have a \"wasi-root\" key", target.triple)
             })
-            .join("lib/wasm32-wasi");
+            .join("lib/wasm32-wasi-preview1");
```

Comment:
Is it able to find wasi-libc sysroot? My understanding wasi-libc targets doesn't mention `-preview1`

Reply:
Also I might be doing something wrong though, but could you try to run it yourself please @yoshuawuyts

Reply:
tried to fix it by removing `-preview1` form here, but it doesn't fix the problem

Reply:
> tried to fix it by removing -preview1 from here, but it doesn't fix the problem

You most likely have to revert the other change in the same file (line 411) as well.

Reply:
Yeah I did, stiil the same